src/foam/comics/InlineDAOControllerView.js

```diff
@@ -47,7 +47,7 @@ foam.CLASS({
         selectedObjects$: this.data.selectedObjects$
       },
       this,
-      this.__subContext__.createSubContext({ memento: null }));
+      this.__subContext__.createSubContext({ memento: this.memento }));
```

Comment:
Isn't this the same as just doing nothing? If we didn't import the memento and then export it here, the memento in the context would just flow through as normal.

Reply:
It is not. This view does not have the memento in its subcontext. Not sure why that is.

Reply:
Why are you creating the view and adding that rather than just adding the viewSpec?
If you keep the this.__subContext__ but remove the createSubContext(), does it work?

Reply:
It does not. 

Not sure why we started using VIewSpec but it happened in https://github.com/kgrgreer/foam3/commit/d57d986b4c1ec16125fc31188392d07f42e3743d

Reply:
Can we put it back to using the viewspec? Then if something breaks again, we'll know, and can document it.

Reply:
Done.